math/dec_test.go

```
@@ -1,1070 +1,1412 @@
-	type decimalTest []string
-	var testcases []decimalTest
-	raw, err := os.ReadFile("./testdata/decimals.json")
-	require.NoError(t, err)
-	err = json.Unmarshal(raw, &testcases)
-	require.NoError(t, err)
-
-	for _, tc := range testcases {
-		t.Run(tc[0], func(t *testing.T) {
-			out, err := math.FormatDec(tc[0])
+	for _, tc := range tcs {
+		t.Run(fmt.Sprintf(tc.src), func(t *testing.T) {
```

golangci-lint

printf: non-constant format string in call to fmt.Sprintf

(govet)